lib/pbench/server/api/resources/__init__.py

```diff
@@ -337,27 +351,36 @@ def convert_keyword(value: str, parameter: "Parameter") -> str:
     raise KeywordError(parameter, "keyword", [value])
 
 
-def convert_list(value: List[Any], parameter: "Parameter") -> List[Any]:
+def convert_list(value: JSONVALUE, parameter: "Parameter") -> List[Any]:
```

Comment:
It seems like the type hint for `value` should be `Union[JSONARRAY, JSONSTRING]` or `Union[List[Any], str]` -- this function won't accept any other type of `JSONVALUE`.

Reply:
Very possibly. ☹️